src/partials/social.html

```diff
@@ -26,7 +26,7 @@
     <link rel="stylesheet" type="text/css"
         href="{{ 'assets/fonts/font-awesome.css' | url }}" />
     {% for social in config.extra.social %}
-      <a href="{{ social.link }}" class="md-footer-social__link
+      <a href="{{ social.link }}" rel="noopener" title="{{ social.type }}" class="md-footer-social__link
```

Comment:
If a link here doesn't open in a new tab/page then it doesn't need rel="noopener"

Reply:
I also remember it being a mitigation for `target=_blank`

Reply:
Oh indeed, I have external links set to open in a new window in my fork, so I guess I can drop that here.

Reply:
@squidfunk I remember you not being the biggest fan of `target="_blank"` but I think it makes sense for social links because of tracking etc, would you be open to adding that to just the social partial then?

Reply:
The reason why we removed it is [here](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/). I guess we could add it back to the social links. I don't know. Does somebody else has an opinion on this?

Reply:
That concern is fixed by also having noopener set as well though, no?

Reply:
I personally don't use the social links in MkDocs-Material, but if I did I'd prefer them to open in a new tab. So adding `target="_blank" rel="noopener"` to just the social links gets a 👍  from me.

Reply:
Ok, then let's go ahead and add `target=_blank` back. I just wanted another opinion, as I'm comfortable with the current solution, but that may be due to the intrinsic blindness I have when I see a project using this theme.

Reply:
Alright, added! What about the footer links? Remove noopener?

Reply:
Well, to be consistent, I would also add `target=_blank` + `noopener` back to the footer links.